cypress/integration/models/administrator/proxy/proxy.ts

```diff
@@ -56,6 +57,15 @@ export class Proxy {
         submitForm();
     }
 
+    protected configureInvalidProxy(type: string): void {
+        cy.wait(5000);
+        this.enableSwitch(`#${type}Proxy`);
+        inputText(`[name="${type}Host"]`, this.hostname);
+        inputText(`[name="${type}Port"]`, "Invalid port");
+        cy.get("#port-helper").contains("This field is required");
+        cy.get(submitButton).should("be.disabled");
```

Comment:
Models can only contain methods for valid operations. These steps will be part of tests in proxy.test.ts.